### PR TITLE
Feat: Added New Phones

### PR DESCRIPTION
**Pull request by UniqueDevStorm**
# Phones
## Apple
- iPhone 12 Pro Max
- iPhone 12 Pro
- iPhone 12
- iPhone 12 mini
## Samsung
- Z Filp
- Fold
- Fold 2
- Galaxy S20 FE
- Galaxy S21
- Galaxy S21+
- Galaxy Note20
- Galaxy Note20 Ultra

# Colors
28 New Color
## Phantom
- Phantom Violet
- Phantom Pink
- Phantom White
- Phantom Grey
- Phantom Silver
- Phantom Black
- Phantom Gold
- Phantom Red
- Phantom Titanium
- Phantom Navy
- Phantom Brown

## Cloud
- Cloud Red
- Cloud Orange
- Cloud Lavender
- Cloud Navy
- Cloud Mint

## Cosmos
- Cosmos Black

## Mirror
- Mirror Black
-Mirror Purple
- Mirror Gold

## Mystic
- Mystic Grey
- Mystic Bronze
- Mystic White
- Mystic Black
- Mystic Green

## Pacific
- Pacific blue

- Graphite
- Space Silver

### CODE REVIEW

**Comment by wonderlandpark** [comment]
Thx for the PR.

I will review it soon

File: src/getPhone.js [hunk 1]
```diff
@@ -5,15 +5,14 @@ export function getDropdown(company) {
...
     return [];
   } else {
-    var g = phones[company].map(el => {
+    return phones[company].map(el => {
       return {
         key: el.id,
         value: el.id,
         text: `${el.name} (${el.date} 출시)`,
-        image: { src: `/phones/${el.id}.jpg` }
+        image: {src: `/phones/${el.id}.jpg`}
```
Comment (wonderlandpark) [inline src/getPhone.js:13]: Better looking
```suggestion
        image: { src: `/phones/${el.id}.jpg` }
```

File: src/phone.json [hunk 2]
```diff
@@ -454,6 +576,309 @@
...
+      "camera": {
+        "front": ["40MP"],
+        "back": [
+          "기본: 12.0MP(OIS 지원)",
+          "광각: 12.0MP"
+        ]
+      }
+    },
+    {
+      "id": "zfilp",
```
Comment (wonderlandpark) [inline src/phone.json:852]: typo
```suggestion
      "id": "zflip",
```

**Review by wonderlandpark — CHANGES_REQUESTED** [review]
Looks Good 👀

File: src/phone.json (hunk 2 above)
Comment (UniqueDevStorm) [reply src/phone.json:852]: > typo

Oops, There was typo

File: src/getPhone.js (hunk 1 above)
Comment (wonderlandpark) [reply src/getPhone.js:13]: Why are you resolving this?

File: src/phone.json (hunk 2 above)
Comment (wonderlandpark) [reply src/phone.json:852]: Why are you resolving this?

**Comment by wonderlandpark** [comment]
@UniqueDevStorm Could you commit my suggestion? I could merge it right now.

**Review by wonderlandpark — CHANGES_REQUESTED** [review]
Looks like Note 20 5G and zFlip 5G is missing image 😢

**Comment by UniqueDevStorm** [comment]
> Looks like Note 20 5G and zFlip 5G is missing image 😢

The name has been changed, but the image value seems to have not changed.

**Comment by wonderlandpark** [comment]
image
Could you double-check it? Looks like it's not fixed yet.

**Comment by UniqueDevStorm** [comment]
https://phonepare-1.vercel.app
I'm done fixing it.

**Review by wonderlandpark — APPROVED** [review]

**Comment by wonderlandpark** [comment]
Thanks for your contribution

**Comment by UniqueDevStorm** [comment]
> Thanks for your contribution

I will update you when another smartphone comes out next time.

**Pull request merged**